swift/actions/database-upgrade-scripts/action.yml

```
@@ -0,0 +1,23 @@
+name: Check Swift database upgrade/downgrade scripts
+runs:
+  using: composite
+  steps:
+    - name: Check upgrade scripts
+      shell: bash
+      working-directory: swift
+      run: |
+        echo > empty.trap
+        codeql dataset import -S ql/lib/upgrades/initial/swift.dbscheme testdb empty.trap
+        codeql dataset upgrade testdb --additional-packs ql/lib
+        diff -q testdb/swift.dbscheme ql/lib/swift.dbscheme
+    - name: Check downgrade scripts
+      shell: bash
+      working-directory: swift
+      run: |
+        echo > empty.trap
+        rm -rf testdb
+        codeql dataset import -S ql/lib/swift.dbscheme testdb empty.trap
+        codeql resolve upgrades --format=lines --allow-downgrades --additional-packs downgrades \
+         --dbscheme=ql/lib/swift.dbscheme --target-dbscheme=downgrades/initial/swift.dbscheme |
+         xargs codeql execute upgrades testdb
```

This will make `xargs` not run on empty
```suggestion
         xargs -r codeql execute upgrades testdb
```